.github/workflows/ubuntu-packaging.yml

```diff
@@ -72,39 +60,33 @@ jobs:
           apt-get install -y ./build/pack-gcc-${{matrix.build_type}}/*.deb
 
       - name: Upload debian package
-        if: matrix.architecture == 'x86_64'
         uses: actions/upload-artifact@v3
         with:
           name: tfc-framework-${{matrix.build_type}}
-          path: build/pack-gcc-${{matrix.build_type}}/*.deb
+          path: build/pack-*/*.deb
+
       - name: Upload rpm package
-        if: matrix.architecture == 'x86_64'
         uses: actions/upload-artifact@v3
         with:
           name: tfc-framework-${{matrix.build_type}}
-          path: build/pack-gcc-${{matrix.build_type}}/*.rpm
+          path: build/pack-*/*.rpm
+
       - name: Upload tar package
-        if: matrix.architecture == 'x86_64'
         uses: actions/upload-artifact@v3
         with:
           name: tfc-framework-${{matrix.build_type}}
-          path: build/pack-gcc-${{matrix.build_type}}/*.tar.gz
+          path: build/pack-*/*.tar.gz
 
-      - name: Upload debian package
-        if: matrix.architecture == 'aarch64'
-        uses: actions/upload-artifact@v3
-        with:
-          name: tfc-framework-${{matrix.build_type}}
-          path: build/pack-aarch64-gcc-${{matrix.build_type}}/*.deb
-      - name: Upload rpm package
-        if: matrix.architecture == 'aarch64'
-        uses: actions/upload-artifact@v3
-        with:
-          name: tfc-framework-${{matrix.build_type}}
-          path: build/pack-aarch64-gcc-${{matrix.build_type}}/*.rpm
-      - name: Upload tar package
-        if: matrix.architecture == 'aarch64'
-        uses: actions/upload-artifact@v3
+      - name: Cloudsmith upload
```

Comment:
should we restrict to only run on master?

Reply:
That kind of depends on how we design our release structure, I was actually sitting on a local commit that restricts this to relases. And then my thougt was we would create a tag.

Reply:
For now I would say main is sufficient, but tags would be even better, we can obviously start tagging main with small incremental releases

Reply:
Let's do that. I will create a tagged inital release after this is merged as 2023.10.1